test.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'Person'
```

Comment:
- [ ] [OPTIONAL] Your file name is person.rb so you should use lowercase person for require_relative, currently you have used capital Person and this will throw an error saying it cannot load such file